tests_end_to_end/application_sanity/conftest.py

```diff
@@ -30,6 +37,42 @@ def client(config):
     return opik.Opik(project_name=config['project']['name'], host='http://localhost:5173/api')
 
 
+@pytest.fixture(scope='function')
+def projects_page(page: Page):
+    projects_page = ProjectsPage(page)
+    projects_page.go_to_page()
+    return projects_page
+    
+
+@pytest.fixture(scope='function')
+def projects_page_timeout(page: Page):
+    projects_page = ProjectsPage(page)
+    projects_page.go_to_page()
+    projects_page.page.wait_for_timeout(7000)
```

Comment:
We need to figure it out how to remove these timeouts but for now it's okay to keep them